src/vs/workbench/parts/debug/common/debugModel.ts

```diff
@@ -414,6 +424,14 @@ export class Model extends ee.EventEmitter implements debug.IModel {
 		this.emit(debug.ModelEvents.CALLSTACK_UPDATED);
 	}
 
+	public continueThreads(): void {
```

Comment:
I could add the functionality of `continueThreads` to `clearThreads` behind yet another boolean arguments.

I think having an explicit method is better (`clearThreads` is already confusing enough). Let me know what you think.